package/views.py

```diff
@@ -17,7 +17,7 @@
 from django.views.generic.base import TemplateView
 from django_q.tasks import async_task
 from django_tables2 import SingleTableView
-from settings.base import DOCS_URL
+from core.settings import DOCS_URL
```

Comment:
To avoid having to change this in the future, this should really be: 

```suggestion
from django.conf import DOCS_URL
```